client/src/components/app.jsx

```diff
@@ -1,24 +1,48 @@
 /* eslint-disable no-unused-vars */
 import React from 'react';
+import axios from 'axios';
 import Overview from './overview/index.jsx';
 import QandA from './qa/QandA.jsx';
 import RelatedProducts from './relatedItemsAndComparison/RelatedProducts.jsx';
 import YourOutfitList from './relatedItemsAndComparison/YourOutfitList.jsx';
 import examples from '../examples.js';
 
 function App() {
-  const [product, setProduct] = React.useState(examples.product);
-  const [styles, setStyles] = React.useState(examples.styles.results);
-  const [meta, setMeta] = React.useState(examples.meta);
+  const [product, setProduct] = React.useState(null);
+  const [styles, setStyles] = React.useState(null);
+  const [meta, setMeta] = React.useState(null);
+  // eslint-disable-next-line no-undef
+  const seedPID = +document.querySelector('main').getAttribute('pid');
 
-  return (
+  React.useEffect(() => {
+    Promise.all([
```

Comment:
I love the idea of promisifying all the axios calls, that's what I was trying to do in the product card. It didn't work for me, because I think I missed the conditional for return. 

Do you think we should move all the axios calls from other widgets in here as well and do the same with the states and props what you've done for your widget?